src/components/WelcomeIntro.js

```diff
@@ -1,15 +1,93 @@
 import React from "react";
-import { Col, Container, Row } from "react-bootstrap";
+import { Col, Container, Row} from "react-bootstrap";
+import CocoaBean from "../assets/CocoaBean.svg";
+import JellyBean from "../assets/JellyBean.svg";
+import GardenBean from "../assets/GardenBean.svg";
+import SoyBean from "../assets/SoyBean.svg";
+import MagicBean from "../assets/MagicBean.svg";
+import CabinCupLogo from "../assets/CabinCupLogo.svg";
+
+import useAirtableAPI from "../hooks/api-hook";
+
+
 
 const WelcomeIntro = () => {
+  const { data, isLoading } = useAirtableAPI("appLHUnzVRxpj7Nx1", "Raffle");
+  console.log(data);
+  var cabinPoints = {
+    "Jelly Beans": 0,
+    "Cocoa Beans": 0,
+    "Soy Beans": 0,
+    "Garden Beans": 0,
+    "Magic Beans": 0,
+  }
+
+  data.forEach(data => {
+    var cabin = data.fields["Cabins"];
+    cabinPoints[cabin] += 1;
+  })
+  console.log(cabinPoints);
   return (
-    <Container className="mt-5">
+    <Container>
       <Row>
-        <Col>
-          <div className="cabin">Cabin Intro</div>
+        <Col md={6}>
+          <div className="cabin-cup">
+            <Row>
+              <Col md={{span: 7, offset: 1}}>
+                <div className="cabin-cup-intro">
+                <h2>Cabin Cup</h2> 
+                  <p>
+                  Earn points for your cabin by bonding with you cabin mates 
+                  and participating in cabin events! Use your points in the end 
+                  to enter raffles for awesome prizes! 
+                 </p>
+                </div>
+                <br/>
+              </Col> 
+              <Col className= "cabin-logo" md={{span: 3, offset: 1}}>
+              <img src={CabinCupLogo} alt="Cabin cup logo"/>
+              </Col>
+            </Row>
+            <Row>
+              <Col md={{span: 3.5, offset: 3}}>
+              <p className= "beanPoints">
+              {cabinPoints["Jelly Beans"]}
```

Comment:
Raquel had suggested adding "pts" after the numbers so it's clear that they're the number of points. Could we add that in here as well?